### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by klduran22**
Hello. I may be 2 years late but I am just taking a matlab class and I discovered this...so I wanted to give it a go? Thanks.

### CODE REVIEW

**Comment by scottclowe** [comment]
You're not too late at all. However, you've not issued the pull request correctly. At the moment you are requesting to merge from your branch, klduran22/matlab-schemes/klduran22-patch-1, into your other branch, klduran22/matlab-schemes/master. Instead, you need to request a merge into the upstream repository (my repository) scottclowe/matlab-schemes/master.

You should follow the instructions here: https://help.github.com/articles/creating-a-pull-request-from-a-fork/

Basically you need to click the New Pull Request button on this page here: https://github.com/scottclowe/matlab-schemes/pulls, and then select your klduran22-patch-1 branch.

(You're actually rather lucky I happened to spot this!)

**Comment by klduran22** [comment]
Oops my bad, I do not know how to work GitHub.
